README.md

```diff
@@ -43,7 +46,7 @@ namespace MyNamespace
 }
 ```
 
-#### To support your mod for the QMods system, you need to learn how the `mod.json` file works. It contains informations about a mod, and it can have the folling keys:
+**To support your mod for the QMods system, you need to learn how the `mod.json` file works. It contains informations about a mod, and it can have the folling keys:**
```

Comment:
Include information about attributes and such

Reply:
That information is on the wiki now.  
Is it still needed in the README.md?